packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/relay/relay.ts

```diff
@@ -11,22 +10,18 @@ import { Request, Response, NextFunction } from 'express'
 import { config } from '../../config'
 import { BadRequestError } from '../../errors'
 import { assertRelayAllowedInstructions } from './assertRelayAllowedInstructions'
-import { cacheTransaction, getCachedDiscoveryNodes } from '../../redis'
-import fetch from 'cross-fetch'
+import {} from 'cross-fetch'
```

Comment:
should be `fetch` not `{}` right?

Reply:
fetch itself actually isn't used but if i don't import this I get lint errors with the signal code down here

```
  return new Promise<void>((resolve, reject) => {
    if (signal?.aborted) {
      reject()
    }
    const listener = () => {
      clearTimeout(timer)
      reject()
    }
    const timer = setTimeout(() => {
      signal?.removeEventListener('abort', listener)
      resolve()
    }, ms)
    signal?.addEventListener('abort', listener)
  })
```

Reply:
that's... super odd!